src/engraving/playback/renderers/chordarticulationsrenderer.cpp

```diff
@@ -126,7 +126,7 @@ void ChordArticulationsRenderer::renderNote(const Chord* chord, const Note* note
     NoteArticulationsParser::buildNoteArticulationMap(note, ctx, noteCtx.chordCtx.commonArticulations);
 
     if (note->tieFor()) {
-        noteCtx.duration = tiedNotesTotalDuration(note->score(), note, noteCtx.duration);
+        noteCtx.duration = tiedNotesTotalDuration(note->score(), note, noteCtx.duration, ctx.positionTickOffset);
```

Comment:
Could you please add a new unit test or update an existing one?

Reply:
Done! Verified that this test fails without these changes and passes with them.